Source/WebCore/dom/Node.cpp

```diff
@@ -115,10 +115,10 @@ static const char* stringForRareDataUseType(NodeRareData::UseType useType)
         return "NodeList";
     case NodeRareData::UseType::MutationObserver:
         return "MutationObserver";
+    case NodeRareData::UseType::ManuallyAssignedSlot:
```

Comment:
Is Is "manually assigned" a spec term? If not then fow about "ImperativelyAssigned" or something (here and everywhere else)? I don't think "manual" is a good way to describe something done via API.

Reply:
Yes, manually assigned is a spec term:
https://html.spec.whatwg.org/multipage/scripting.html#manually-assigned-nodes
I'd add a link to this spec in the header.